pkg/kubelet/kubelet.go

```diff
@@ -920,10 +926,20 @@ func (kl *Kubelet) listPodsFromDisk() ([]types.UID, error) {
 // Starts garbage collection threads.
 func (kl *Kubelet) StartGarbageCollection() {
```

Comment:
StartGarbageCollection() and Run() are both methods in KubeletBootstrap interface. Changing how they interact seems to be outside the scope of this PR,